# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

sensor_msgs/Image image
geometry_msgs/Pose2D pose
